### ISSUE TITLE
ActionController::UrlGenerationError in Members::Dashboard#index

### ISSUE THREAD

**Issue by dan-mi-sun** [blockades/mmt#20]
Running the following commands on `development`

```
 48  git checkout development
   49  git pull
   50  bundle exec db:create
   51  bundle install
   52  bundle exec db:create
   53  bundle install
   54  bundle exec db:create
   55  bundle exec rake db:create
   56  bundle exec rake db:migrate
   57  bundle exec rake setup:coins
   58  rails s
   59  rails db:seed
   60  rails s
```

And getting the following:

---

Showing /home/user/Projects/mmt/app/views/layouts/application.html.slim where line #13 raised:

No route matches {:action=>"show", :controller=>"members/portfolios", :id=>nil}, missing required keys: [:id]

around line 13
```
    nav.primary
      ul.left
        li= link_to "Portfolio", portfolio_path(current_member.live_portfolio), class: "button button-primary"
        li= link_to "My Profile", member_path(current_member), class: "button button-primary"
      ul.right
        - if current_member&.admin?

```

**Comment by KGibb8** [comment]
run 

`rake setup:coins && rake setup:portfolios`

**Comment by KGibb8** [comment]
See above PR for proper fix. Please merge!

**Comment by dan-mi-sun** [comment]
Ok, Have done so and am getting the following:

```
[  9:26am ]  [ user@host:~/Projects/mmt(feature/event_store_5th_iteration✔) ]
 $ bundle exec rake setup
rake aborted!
LoadError: Unable to autoload constant Transaction::SystemAllocation, expected /home/user/Projects/mmt/app/subscribers/transaction/system_allocation.rb to define it
/home/user/Projects/mmt/config/initializers/wisper.rb:3:in `<top (required)>'
/home/user/Projects/mmt/config/environment.rb:7:in `<top (required)>'
/home/user/.rbenv/versions/2.4.0/bin/bundle:22:in `load'
/home/user/.rbenv/versions/2.4.0/bin/bundle:22:in `<main>'
Tasks: TOP => setup => setup:all => setup:members => environment
(See full trace by running task with --trace)
```

**Comment by KGibb8** [comment]
See https://github.com/blockades/mmt/issues/19

**Comment by dan-mi-sun** [comment]
I am closing this as it seems to have been fixed with #19
